spyderlib/config.py

```diff
@@ -270,7 +270,7 @@ def is_ubuntu():
               'font/italic': False,
               'font/bold': False,
               'show_banner': True,
-              'use_gui_completion': True,

```

Comment:
Given that you're removing an option, you need to bump `CONF_VERSION` below.

See the comment above it to learn how to that :-)